_co-op-companies/N-varian.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Varian Medical Systems
```

Comment:
I think we're missing the closing double quote `"`

Reply:
Done